src/app/pages/overview/overview.component.html

```diff
@@ -0,0 +1,3 @@
+<app-layout bgColor="green">
```

Comment:
What if UX decides to change representation and add for example one more component in the middle of each child page (description, for example)? you will need to add it to all child components, so better to have one component for each child-page ( as they differs only by color and H2 text), you can just use 1 and provide the text through another @Input